package.json

```diff
@@ -1,5 +1,14 @@
 {
     "dependencies": {
-      "cosmiconfig": "8.0.0"
+        "cosmiconfig": "8.0.0"
+    },
+    "devDependencies": {
+        "@commitlint/types": "^17.4.4",
```

Comment:
@Mersho the packages being removed from the `add --dev` call are: jest typescript ts-jest @types/jest ts-node, NOT THIS ONE!